conan/tools/cmake/cmake.py

```diff
@@ -193,6 +196,14 @@ def install(self, build_type=None, component=None):
         if do_strip:
             arg_list.append("--strip")
 
+        if cli_args:
+            if "--install" in cli_args:
+                raise ConanException("Do not pass '--install' argument to 'install()'")
+            if "--strip" in cli_args:
```

Comment:
I am not sure about this warning, it should either be normally allowed or raise an error, but from my experience this kind of warning doesn't help much as it pass unnoticed, or it annoys the user that actually wants to do that.

Reply:
Sure :) The rationale for that warning was that I wanted the user to be aware of that feature. It wasn't even documented, and I got to know about it myself while reading the code.

I removed the warning. The configuration mentioning in the docs is added separately at https://github.com/conan-io/docs/pull/3314